src/script.js

```diff
@@ -1,26 +1,17 @@
- function convertGrossToNet() {  
-    // Lấy giá trị thu nhập từ input  
-    const incomeInput = document.getElementById('incomeInput').value; 
-     
-    // Lấy phần tử bảng cho các ô hiển thị kết quả (Hàng thứ 2, cột thứ 1)
-    const grossCell = document.querySelector('table tr:nth-child(2) td:nth-child(1)');  
-    
-    // Cập nhật giá trị Lương Gross trong bảng  
-    grossCell.textContent = incomeInput || 0; // Nếu không có giá trị, hiển thị 0  
-}  
+import grossToNet from "./salary-app/grossToNet.js";
 
-// Ngăn không cho form tự động gửi và tải lại trang khi nút được nhấn  
-function preventFormSubmission(event) {  
-    event.preventDefault();  
-}  
+console.log(0.1);
 
-// Hàm gắn sự kiện khi trang được tải  
-window.onload = function() {  
-    // Thêm sự kiện click cho nút Gross sang Net  
-    document.getElementById('grossToNetButton').addEventListener('click', function(event) {  
-        preventFormSubmission(event); // Ngăn gửi form  
-        
-        this.classList.toggle("style-selected-button")
-        convertGrossToNet(); // Gọi hàm để chuyển đổi  
-    });  
-};
\ No newline at end of file
+document.addEventListener('DOMContentLoaded', function () {
+    console.log('DOM fully loaded and parsed');
+    document.getElementById("btnGrossToNet").addEventListener("click", async function () {
+        const income = document.getElementById("income").value; // Lấy dữ liệu từ input
+        const result = await grossToNet(income); // Gọi hàm tính toán
+        const LOCALE = "vi-VI";
```

Comment:
this should be outside, init it per btn click is redundant. 

Same for the next 4 dom elements